ta/crypt/arith_taf.c

```diff
@@ -2,6 +2,7 @@
 /* Copyright (c) 2018, Linaro Limited */
 
 #include <arith_taf.h>
+#include <inttypes.h>
```

Comment:
I don't expect that this file is compiled with musl-libc.
I agree with the change, it's just that the commit message isn't accurate for this file.

Reply:
You're right. I've updated the subjects of all related patches